src/myjarvis/domain/entities/chat_context.py

```
@@ -1,17 +1,213 @@
+from src.myjarvis.domain.services import MessageOperationsService
+from src.myjarvis.domain.value_objects import (
+    Message,
+    ChatLimits,
+    MessageCollection,
+)
+
+ErrorsMessages: TypeAlias = Optional[List[str]]
+
+
+class ChatContext:
+    def __init__(
```

Pylint (3.3.7)

[refactor] Too many arguments (9/5)

(R0913)